crates/biome_html_analyze/tests/specs/a11y/useIframeTitle/invalid.html

```diff
@@ -0,0 +1,5 @@
+<!-- should generate diagnostics -->
+<iframe />
+<iframe></iframe>
+<iframe title></iframe>
+<iframe title=""></iframe>
```

Comment:
As hinted by the rule, tags can be uppercase. So we need to test against e.g. `<IFRAME>`

Also, we need to make sure that if we are inside HTML-ish languages, we don't trigger the rule **for now**, *or we match against only `iframe`*.

At the moment we aren't able to check if a tag is a component, so we don't know  if `Iframe` is a component or not, so we need to be lax in these files.

Let's add tests for these cases, and update the docs to highlight the fact that outside of `.html` files, we only check **exactly** for `iframe`